### PR TITLE
Ready to use in Symfony3

### PR DESCRIPTION
**Pull request by terox**
Hello!

I have changed some files to do this bundle compatible with Symfony2/3.

### CODE REVIEW

**Comment by kieljohn** [comment]
Hi Terox!

Thanks for your contribution, our team will take a look at this and try to get it merged

File: src/Vivait/StringGeneratorBundle/Generator/SecureBytesGenerator.php [hunk 1]
```diff
@@ -37,7 +28,7 @@ public function setLength($length)
      */
     public function generate()
     {
-        return $this->secureRandom->nextBytes($this->length);
+        return random_bytes($this->length);
```
Comment (leightonthomas) [inline src/Vivait/StringGeneratorBundle/Generator/SecureBytesGenerator.php]: This only seems to be available in PHP 7 - perhaps https://github.com/symfony/polyfill-php70 could be used?

**Comment by leightonthomas** [comment]
`.travis.yml` will need updating due to the PHP version requirement change and I think there's some phpspec test issues that need resolving as well

File: composer.json [hunk 2]
```diff
@@ -11,11 +11,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5.3",
         "doctrine/common": "~2.2",
         "doctrine/orm": "~2.2",
-        "symfony/security": "~2.1",
-        "symfony/options-resolver": "~2.1",
+        "symfony/security": "~2.8|~3.2",
```
Comment (Brunty) [inline composer.json]: If this is to support Symfony 3 as well, then what about 3.0 and 3.1?

I'd suggest a change to: `^2.8|^3.0`

File: composer.json [hunk 3]
```diff
@@ -11,11 +11,11 @@
...
+        "symfony/options-resolver": "~2.8|~3.2",
```
Comment (Brunty) [inline composer.json]: Same requirements change on this as the `symfony/security`

File: src/Vivait/StringGeneratorBundle/Generator/SecureBytesGenerator.php (hunk 1 above)
Comment (Brunty) [inline src/Vivait/StringGeneratorBundle/Generator/SecureBytesGenerator.php]: However, I think if we're going to rely on `random_bytes`, we need to be explicit in the polyfill package as a dependency of ours.

We can't necessarily rely on the security component including it.

**Comment by Brunty** [comment]
Nice work, thanks @terox!

**Comment by terox** [comment]
Please, could you check the changes? I removed the PHP 5.4 from ```.travis.yml``` because the mininum requeriments for Symfony 2.8 and Symfony 3 I think that is 5.5

**Comment by kieljohn** [comment]
Looks good, thanks for the contribution. We'll merge and tag a version early next week 👍

**Comment by terox** [comment]
Thank you so much for this fantastic bundle ;)

**Pull request merged**